templates/addons/cluster-resource-set/secret.yaml

```diff
@@ -12,6 +12,7 @@ stringData:
       namespace: kube-system
     stringData:
       apiToken: ${LINODE_TOKEN}
+      token: ${LINODE_TOKEN}
```

Comment:
Could we instead update the values on the CSI chart to use apiToken instead of token?

Reply:
updated